op-defender/psp_executor/defender.go

```diff
@@ -0,0 +1,290 @@
+package psp_executor
+
+import (
+	"context"
+	"crypto/ecdsa"
+	"encoding/json"
+	"fmt"
+	"github.com/ethereum-optimism/optimism/op-bindings/bindings"
+	"github.com/ethereum-optimism/optimism/op-service/metrics"
+	"github.com/ethereum/go-ethereum/accounts/abi/bind"
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/core/types"
+	"github.com/ethereum/go-ethereum/crypto"
+	"github.com/ethereum/go-ethereum/ethclient"
+	"github.com/ethereum/go-ethereum/log"
+	"github.com/gorilla/mux"
+	"github.com/prometheus/client_golang/prometheus"
+	"math/big"
+	"net/http"
+)
+
+// **********************************************************************
+// *                        Informations:                               *
+// **********************************************************************
+// * Sepolia:                                                           *
+// * deputyGuardianSepolia: 0x4220C5deD9dC2C8a8366e684B098094790C72d3c *
+// * SuperChainConfigSepolia: 0xC2Be75506d5724086DEB7245bd260Cc9753911Be *
+// * FoS on Sepolia: 0x837DE453AD5F21E89771e3c06239d8236c0EFd5E        *
+// **********************************************************************
+// * Mainnet:                                                           *
+// * deputyGuardianMainnet: 0x5dC91D01290af474CE21DE14c17335a6dEe4d2a8  *
+// **********************************************************************
+
+const (
+	MetricsNamespace = "psp_executor"
+	SepoliaRPC       = "https://proxyd-l1-consensus.primary.sepolia.prod.oplabs.cloud"
+	MainnetRPC       = "https://proxyd-l1-consensus.primary.mainnet.prod.oplabs.cloud"
```

Comment:
As above

Reply:
Yes all the hardcoded values for now are hardcoded but they will be fetch from configuration in the future.
For now, will keep the constant and hardcoded values and will be removed in the PR 3.